bin/filter_sce.R

```diff
@@ -32,6 +32,12 @@ option_list <- list(
     default = FALSE,
     help = "flag to set the miQC's enforce_left_cutoff option to TRUE"
   ),
+  make_option(
+    opt_str = c("--no_sce_compression"),
+    action = "store_false",
+    default = TRUE,
+    help = "flag to turn off compression of the output SCE object"
+  ),
```

Comment:
I appreciate that this is may be less prone to user error, but I would just pass along the compression flag and then check that it is valid (one of `c("none", "gz", "bz2", "xz")`)


```suggestion
  make_option(
    opt_str = c("--sce_compression"),
    type = "character",
    default = "bz2",
    help = "compression method for writing SCE as an RDS file."
  ),
```

Reply:
> so is probably not worth the effort, given we are not likely to need the flexibility.

My sense is if we want to change compression in general, there's a fair bit of changes across the code base to implement as `bz2` is hardcoded all over the place. So, I think this is something to update to if/when we end up needing this flexibility